File: Kap10/JavaScript/Listing23/ArrayTest.js
let myArr = ["a", "b", "c", "e", "X", "F"];
console.log("Wert auf Position 1: " + myArr[1]);
myArr.splice(3, 0, ["d"]);
myArr.splice(5, 1);
myArr[5] = "f";
for (let i = 0; i < myArr.length; i++) {
    console.log("Wert pos: " + i + ": " + myArr[i]);
}

File: Kap10/Python/Listing26/ArrayTest.py
myList = ["a", "b", "c", "e", "X", "F"]; 
print("Wert auf Position 1: " + myList[1])
myList.insert(3, "d")
myList.pop(5)
myList[5] = "f";
for i in range(0, len(myList)):
    print("Wert pos: " + str(i) + ": " + myList[i])
